src/views/Player.vue

```diff
@@ -103,7 +103,8 @@ export default {
       hasVideoPlayed: -1, // Three possible values: -1(don't know), 0(didn't play), 1(played)
       sessionId: 1,
       hasPlyrLoaded: false,
-      retention: []
+      retention: [],
+      previousPlayerTime: 0
```

Comment:
This variable is needed to store the `player_time` when the user left the last session. This variable is used when the player has been initialized and it's ready to play.